code/modules/reagents/chemistry/reagents/virology_reagents.dm

```diff
@@ -0,0 +1,241 @@
+
+//////////////////////////////////////////////////////////////////////////////////////////
+					// VIROLOGY REAGENTS
+//////////////////////////////////////////////////////////////////////////////////////
+
+// where all the reagents related to medicine go.
```

Comment:
Huh.

Anyway, //most// of these are basically copy pasta versions of themselves.